configure.ac

```diff
@@ -872,6 +872,96 @@ AM_CONDITIONAL([OPENTHREAD_ENABLE_RAW_LINK_API], [test "${enable_raw_link_api}"
 AC_DEFINE_UNQUOTED([OPENTHREAD_ENABLE_RAW_LINK_API],[${OPENTHREAD_ENABLE_RAW_LINK_API}],[Define to 1 if you want to enable raw link-layer API])
 
 #
+# Software ACK Timeout Logic
+#
+
+AC_ARG_ENABLE(sw_ack_timeout,
```

Comment:
There is something to be said for a developer to be able to successfully navigate the configure options OpenThread makes available.  Towards that end, I agree with Abtin that these options rightfully belong in the openthread-core-default-config.h.  If i was new to Openthread and had to guess where these options might exist, that file would be my first guess.